### PR TITLE
Add Rake task to redirect "settle in the uk" simple smart answer

### PR DESCRIPTION
**Pull request by edwardkerry**
A one-off Rake task to unpublish the `\settle-in-the-uk` simple smart answer and redirect the pages to various replacements.

[Trello](https://trello.com/c/6VT63pcE/2119-3-implement-redirects-from-all-pages-in-a-smart-answer)

There is a quick script in the linked Trello card we can run to curl the paths and make sure this has worked has expected.

### CODE REVIEW

File: lib/tasks/redirect_settle_in_the_uk.rake
Comment (thomasleese) [inline lib/tasks/redirect_settle_in_the_uk.rake:8]: Nice use of `headers: true`, making the code below more readable!

**Review by thomasleese — APPROVED** [review]

**Comment by edwardkerry** [comment]
Closing as the associated work is waiting for long-term content approval.

**Pull request closed without merging**
